packages/core/src/lib/implementation/read-rc-file.unit.test.ts

```diff
@@ -14,7 +14,7 @@ vi.mock('bundle-require', async () => {
       .fn()
       .mockImplementation((options: { filepath: string }) => {
         const extension =
-          options.filepath?.split('.').at(-1) || 'no-extension-found';
+          options.filepath.split('.').at(-1) || 'no-extension-found';
```

Comment:
By looking at the implementation, this should never happen. The function always provides a non-empty extension to `readRcByPath` → `importEsmModule` → `bundle-require`. If the extension is not found, the function throws.

```suggestion
          options.filepath.split('.').at(-1);
```